Source/WebCore/platform/graphics/mac/controls/WebControlView.h

```diff
@@ -37,4 +37,12 @@
 @property (class) NSRect clipBounds;
 @end
 
+@interface NSTextFieldCell ()
```

Comment:
Rather than copy this here, move it to an SPI header.

Reply:
Fixed.